src/bin/tikv-ctl.rs

```diff
@@ -106,7 +106,7 @@ fn new_debug_executor(
             ))) as Box<DebugExecutor>
         }
         (Some(remote), None) => Box::new(new_debug_client(remote, mgr)) as Box<DebugExecutor>,
-        _ => unreachable!(),
+        _ => unreachable!("Missing required parameters: --pd or --db"),
```

Comment:
`unreachable!` panics, what about `eprintln!()` + `process::exit()`?

Reply:
Good idea, let's make sure it exits with a non-zero code.

Reply:
why not using `perror_and_exit`?

Reply:
Is there need to create a custom exception?

Reply:
I think `unreachable` or `panic` ensure that already.